applications/ShapeOptimizationApplication/python_scripts/custom_ios/wrl_io.py

```diff
@@ -0,0 +1,74 @@
+# ==============================================================================
+#  KratosShapeOptimizationApplication
+#
+#  License:         BSD License
+#                   license: ShapeOptimizationApplication/license.txt
+#
+#  Main authors:    Geiser Armin, https://github.com/armingeiser
+#
+# ==============================================================================
+
+import KratosMultiphysics as KM
+from .wrl_reader import read_shapes
+
+
+def _rename_to_valid_name(model_part, shape):
+    name = shape.name
+
+    ref_name = name
+    i = 0
```

Comment:
wouldn't it make more sense to start from 2? So if e.g. "triangle_mesh" is already given, the next shape would be "triangle_mesh_2" and not "triangle_mesh_0"